app/models/spree/return_authorization.rb

```diff
@@ -95,6 +95,7 @@ def process_return
       credit.save
 
       order.return if inventory_units.all?(&:returned?)
+      order.update!
```

Comment:
Return authorizations were adding an adjustment on the order (which updated the order totals automatically via the callback). It needs to be called once after a return is processed, so we can just do it explicitly.